_includes/help/nav_sidenav.html

```diff
@@ -30,6 +31,24 @@
                 {{ question.title }}
               {% endif %}
           </a>
+          {% if question.children.size > 0 %}
+          <ul class="usa-sidenav__sublist">
+              {% for child in children %}
+                {% if question.children contains child.title and question.lang == child.lang %}
+                <li class="usa-sidenav__item">
+                  <a class="{% if child.url == page.url %}usa-current{% endif %}"
+                      href="{{ child.url | prepend: site.baseurl }}">
+                      {% if child.meta_title %}
+                        {{ child.meta_title }}
+                      {% else %}
+                        {{ child.title }}
+                      {% endif %}
+                  </a>
+                </li>
+                {% endif %}
+              {% endfor %}
+            </ul>
```

Comment:
To the prior comment about using URL instead of title, maybe we can adapt the similar `related_questions` logic above to build the children links? This seemed to work for me in some quick local testing.

```suggestion
            {% assign children = site.help | where_exp: 'item', 'item.lang == page.lang and question.children contains item.url' | sort: 'order' %}
            <ul class="usa-sidenav__sublist">
              {% for child in children %}
                <li class="usa-sidenav__item">
                  <a
                    class="{% if child.url == page.url %}usa-current{% endif %}"
                    href="{{ child.url | prepend: site.baseurl }}"
                  >
                    {% if child.meta_title %}
                      {{ child.meta_title }}
                    {% else %}
                      {{ child.title }}
                    {% endif %}
                  </a>
                </li>
              {% endfor %}
            </ul>
```

Reply:
Thank you for the suggestion. Implemented.